modules/surface_matching/samples/ppf_icp.py

```diff
@@ -1,27 +1,32 @@
 import cv2
 import numpy as np
 
+
 def rotation(theta):
     tx, ty, tz = theta
 
-    Rx = np.array([[1, 0, 0], [0, np.cos(tx), -np.sin(tx)], [0, np.sin(tx), np.cos(tx)]])
-    Ry = np.array([[np.cos(ty), 0, -np.sin(ty)], [0, 1, 0], [np.sin(ty), 0, np.cos(ty)]])
-    Rz = np.array([[np.cos(tz), -np.sin(tz), 0], [np.sin(tz), np.cos(tz), 0], [0, 0, 1]])
+    Rx = np.array([[1, 0, 0], [0, np.cos(tx), -np.sin(tx)],
+                   [0, np.sin(tx), np.cos(tx)]])
+    Ry = np.array([[np.cos(ty), 0, -np.sin(ty)],
+                   [0, 1, 0], [np.sin(ty), 0, np.cos(ty)]])
+    Rz = np.array([[np.cos(tz), -np.sin(tz), 0],
+                   [np.sin(tz), np.cos(tz), 0], [0, 0, 1]])
 
     return np.dot(Rx, np.dot(Ry, Rz))
 
+
 width = 20
 height = 10
 max_deg = np.pi / 12
 
-cloud, rotated_cloud = [None]*3, [None]*3
-retval, residual, pose = [None]*3, [None]*3, [None]*3
+cloud, rotated_cloud = [None] * 3, [None] * 3
+retval, residual, pose = [None] * 3, [None] * 3, [None] * 3
 noise = np.random.normal(0.0, 0.1, height * width * 3).reshape((-1, 3))
 noise2 = np.random.normal(0.0, 1.0, height * width)
 
 x, y = np.meshgrid(
-    range(-width/2, width/2),
-    range(-height/2, height/2),
+    range(-width // 2, width // 2),
+    range(-height // 2, height // 2),
```

Comment:
Please eliminate other unnecessary changes. No need to touch unrelated code.
Also proper description for this is: "ppf_icp.py: Python 3 support"

Reply:
/ to // is necessary. Python 3 range requires int not float

Reply:
I am verifying why I needed to make this change to make the code work

Reply:
Although not immediately apparent at a glance, type(-width/2) is float not int

Reply:
You need this change for Python3, but you don't need other 16 changes related to whitespace modifications.